mw-webapp/src/component/notice/Notice.tsx

```diff
@@ -0,0 +1,62 @@
+import {useEffect} from "react";
+import {Button} from "../button/Button";
+import {Title} from "../title/Title";
+import {HeadingLevel} from "../title/Title";
+import styles from "./Notice.module.scss";
+
+const MILLISECONDS = 1000;
+
+export interface NoticeProps {
+  /**
+   * ID for notification, it will be help closing notice by button
+   */
+  id: string;
+  /**
+  * Title for notice (text)
+  */
+  title: string;
+  /**
+   * Description for notice (text)
+   */
+  description: string;
+  /**
+   * Type for notice, like 'Error', 'Ok', 'Warning' (text)
+   */
+  type: string;
+  /**
+   * Timer for automaticle closing (seconds)
+   */
+  timer: number;
+  /**
+   * Callback which close the notification
+   */
+  closeNotice: (id: string) => void;
+}
+
+export const Notice = ({id, title, description, type, timer, closeNotice}: NoticeProps) => {
+
+  const millisecondsTimer = timer * MILLISECONDS;
+
+  useEffect(() => {
+    setTimeout(() => closeNotice(id), millisecondsTimer);
+  }, []);
+  return (
+    <article
```

Comment:
Why tag article?

Reply:
I thougt to write semantic tag. Do you prepare usual "div"

Reply:
No, no div. Just ask is it the best semabtic tag for notifications